core-plugins/widgets/Distinct-batchaggregator.json

```diff
@@ -1,15 +1,25 @@
 {
   "metadata": {
-    "spec-version": "1.0"
+    "spec-version": "1.5"
   },
   "configuration-groups": [
     {
       "label": "Distinct",
       "properties": [
         {
-          "widget-type": "textbox",
+          "widget-type": "csv",
           "label": "Fields",
-          "name": "fields"
+          "name": "fields",
+          "widget-attributes": {
+            "delimiter": ",",
+            "value-placeholder": "Field Name"
+          },
+          "plugin-function": {
+            "method": "POST",
+            "widget": "outputSchema",
+            "output-property": "schema",
+            "plugin-method": "outputSchema"
```

Comment:
do we need `position` and `button-class`?

Reply:
don't know what those are, but it renders fine, and this is what the group by aggregator does.